types/Cargo.toml

```diff
@@ -12,9 +16,11 @@ celestia-proto = { workspace = true }
 const_format = "0.2.31"
 enum_dispatch = "0.3.12"
 cid = { version = "0.10.1", default-features = false, features = ["std"] }
+libp2p = { version = "0.52.3", optional = true }
```

Comment:
use `multiaddr` and `libp2p-identity` crates instead, but still keep them under `libp2p` feature.

Reply:
replaced `libp2p` with `mutliaddr` and `libp2p-identity` in appropriate version :+1: